x-pack/legacy/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_section/privilege_space_form.tsx

```diff
@@ -489,7 +489,7 @@ export class PrivilegeSpaceForm extends Component<Props, State> {
 
       if (
         hasAssignedFeaturePrivileges(form) ||
-        explanation.actualPrivilege === NO_PRIVILEGE_VALUE ||
+        form.base.length === 0 ||
```

Comment:
Without this change, if I gave the user global all and then created a new entry for the default space, the drop-down would default to "All" because this was the "actual privilege" which it inherited globally. Where-as elsewhere we default to "Custom" whenever we can:

![custom](https://user-images.githubusercontent.com/627123/60926847-a88cc700-a25c-11e9-8657-f5c5dba01bec.gif)